vtex-apps/return-app
# V3 items details

#### What problem is this solving?

This PR brings the Product Information table for the return request details page on the admin side. It also adds more information from order items into return request items as `imageUrl`, `unitMultiplier`, `sellerId`, `refId`, `productId` and `sellerName`.

#### How to test it?

We have this branch linked in [this workspace](https://rmav3itemsdetails--powerplanet.myvtex.com/). To test it, one can visit the details page for a request and see the component being used. It's also possible to see the verification status working by updating the request status and verifying the products. Each item has a status as `new`, `denied`, `approved` or `partially approved`

#### Screenshots or example usage:
<img width="1143" alt="Screen Shot 2022-06-14 at 2 39 34 PM" src="https://user-images.githubusercontent.com/38737958/173594541-a28d741c-ef56-4521-87a9-8f21d1773604.png">
<img width="1339" alt="Screen Shot 2022-06-14 at 3 31 00 PM" src="https://user-images.githubusercontent.com/38737958/173594560-79e3cc7c-78b4-44b3-94fe-2832ccaf3083.png">
